### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by yalix13**
cgfc

### CODE REVIEW

**Comment by yalix13** [comment]
hi

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
```
Comment (yalix13) [inline LICENSE:20]: hi 
hqdefault

**Review by yalix13 — COMMENTED** [review]
hi aimbottttttttttttttttttttt

**Comment by yalix13** [comment]
aim bit crazy

**Comment by yalix13** [comment]
Hey this is a craze aim bot that will improve your game
hqdefault

**Comment by yalix13** [comment]
db2416d1d01ceca0085b9783b6e4509599aa1c85

**Comment by HackerHansen** [comment]
?

**Pull request closed without merging**